api_tests/tsconfig.json

```diff
@@ -1,15 +1,25 @@
 {
     "compilerOptions": {
-        "target": "es6",
-        "module": "commonjs",
         "allowSyntheticDefaultImports": true,
-        "noImplicitAny": true,
+        "esModuleInterop": true,
+        "forceConsistentCasingInFileNames": true,
+        "inlineSources": true,
         "lib": ["es5", "es2015", "es2017", "es6", "dom"],
-        "noEmit": true,
-        "noUnusedLocals": true,
-        "noUnusedParameters": true,
+        "module": "commonjs",
+        "moduleResolution": "node",
+        "noEmit": false,
```

Comment:
That should change back to `noEmit` true once you are done.

Reply:
I would need help with this. If I set it to false then TestEnvironment is not transpiled.